public/catalog/v0.1.0/openapi.json

```diff
@@ -342,7 +342,7 @@
             "$ref": "#/components/responses/Forbidden"
           },
           "404": {
-            "description": "Service Offering not found"
+            "description": "Item not found"
```

Comment:
Change below on 416 leaves 'Portfolio Item' as part of the 404.  I think `Service Offering item` would be consistent here.

Reply:
Will update this one, too.